### PR TITLE
Improving UI 

### PR DESCRIPTION
**Pull request by ijosue16**
In this PR I added the following:

- added the styling using CSS and bootstrap.

### CODE REVIEW

**Comment by ijosue16** [comment]
Please note that almost all styling was done during project initialisation, that's why I have a few changes applied 🙏.

File: src/components/Navbar.css [hunk 1]
```diff
@@ -9,3 +9,9 @@
 .icon {
   width: 50px;
 }
+
+.user-account {
+  width: 34px;
+  height: 50px;
+  color: #0290ff;
+}
```
Comment (roshan-bajgain) [inline src/components/Navbar.css:17]: - [x] According to the requirement you should use the exact styling please use the same color. See below for reference
- [x] Also increase the font weight so it matches the design
### Your design
yours

### Original design
fimga

File: src/index.css [hunk 2]
```diff
@@ -14,7 +14,7 @@ body {
     sans-serif;
   -webkit-font-smoothing: antialiased;
   -moz-osx-font-smoothing: grayscale;
-  height: 100vh;
+  height: 100%;
   background-color: #f3f4f8;
 }
```
Comment (roshan-bajgain) [inline src/index.css:19]: - [x] According to the project requirement you should use the exact design as provided. please make your `completed section` in the middle. and also use the same color and font for the `current chapter` section as well. See the image below for reference.
### Your design
his

### Original design
orignial

Comment (roshan-bajgain) [inline src/index.css:19]: - [x] Please stick to the design as much as possible. Please use the same color for buttons and titles. See the image below for reference.
### Your design
ss


###Original deisgn
jj

**Review by roshan-bajgain — CHANGES_REQUESTED** [review]
Hi @ijosue16, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

## Highlights
- No linter errors. ✔️
- GitHub flow is followed. ✔️
- Professional README file. ✔️

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by algerina — APPROVED** [review]
Hi @ijosue16 ,

Good job so far! 

May I highlight the following:

- The repo looks professional.:heavy_check_mark:
- Your project looks awesome:heavy_check_mark:
- You implemented the changes required by the previous TSE:heavy_check_mark:




## STATUS: APPROVED :muscle::clap:

Congratulations! :tada:

Your project is complete!  It's time to merge it :shipit:



 
 ## Optional suggestions

 
 - Please check the comment under the review.

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._
Cheers and Happy coding! :clap: :clap: :clap:

Feel free to leave any questions or comments in the PR thread and tag me @algerina if something is not 100% clear.

#### _**Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.**_
------------------------------------------------------------------------------------------------------------------------------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using this [form.](https://airtable.com/shrQAqnBwek5a0O0s)
___
_If anything seems to be unclear to you, reach out and ask anything by commenting on this Pull Request and mention me as [Algerina](https://github.com/algerina), I will receive a notification and will respond._

[GitHub](https://github.com/algerina) | [LinkedIn](https://linkedin.com/in/amel-khiri) | [Twitter](https://twitter.com/AmelQahwadji)

_Reviewed by [Amel Khiri ](https://github.com/algerina)_

**Pull request merged**